C-Plus-Plus/cp/Reverse_string_using_stack.cpp

```diff
@@ -0,0 +1,58 @@
+/* 
+Given a string,print reverse of the string using stack
+Approach:-
+In stack,we know LIFO principle works.
+First,i create an empty stack,then i push each character of string in stack
+then,using for loop i popped out every single character.
+*/
+
+#include<iostream>
+#include<string>
+#include<stack>
```

Comment:
Use `#include<bits/stdc++.h>` it will save your extra line instead of all three header files.
![Capture](https://user-images.githubusercontent.com/67387053/111437436-1b845b80-8729-11eb-908e-196a61089215.JPG)